erc20-connector/test/TokenLocker.js

```diff
@@ -68,6 +74,110 @@ contract('TokenLocker', function ([addr, addr1]) {
         expect(fromWei(newReceiverBalance) - fromWei(receiverBalance)).equal(1);
     });
 
+    describe('Pausability', () => {
+        it('Lock method', async function() {
+            const amountToTransfer = toWei('1');
+
+            // Let's try to lock some tokens
+            // The balance should be changed and the `Locked` event emitted
+            const preBalance1 = await this.token.balanceOf(addr1);
+            expect(fromWei(preBalance1)).equal(fromWei(initialBalanceAddr1));
+            await this.token.approve(this.locker.address, amountToTransfer, { from: addr1 });
+            const tx1 = await this.locker.lockToken(this.token.address, toWei('1'), 'receiver', { from: addr1 });
```

Comment:
```suggestion
            const tx1 = await this.locker.lockToken(this.token.address, amountToTransfer, 'receiver', { from: addr1 });
```

Reply:
oops, copy-pasted this part from the old test.